pkg/fleets/controller.go

```diff
@@ -408,10 +408,10 @@ func (c *Controller) recreateDeployment(fleet *agonesv1.Fleet, rest []*agonesv1.
 func (c *Controller) rollingUpdateDeployment(fleet *agonesv1.Fleet, active *agonesv1.GameServerSet, rest []*agonesv1.GameServerSet) (int32, error) {
 	replicas, err := c.rollingUpdateActive(fleet, active, rest)
 	if err != nil {
-		return replicas, err
+		return 0, err
```

Comment:
Curious, what's the impetus for this change?

Reply:
left a comment here
https://github.com/googleforgames/agones/pull/1531#discussion_r434166303